docs/DeveloperGuide.adoc

```diff
@@ -806,27 +806,71 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |tutor |list schedule I have in certain month |check and prepare myself in advance
+
+|`* * *` |tutor |add a new tuition schedule |so that I can compile my tuition schedule
+
+|`* * *` |user |delete a schedule in the calendar |update my calender
+
+|`* * *` |tutor |tag the students with their current education level |adjust my teaching accordingly
+
+|`* * *` |tutor |edit student's details |stay updated
+
+|`* * *` |user |edit a schedule |update my calendar
+
+|`* * *` |user |open the calender |check on my schedule
+
+|`* * *` |user |submit feedback to the developers of the application |this application can be further improved
+
+|`* * *` |user |add my personal schedule to the calendar |prevent any schedule from clashing
+
+|`* * *` |tutor |categorize a contact as my student |separate my student from my personal contact lists
+
+|`* *` |tutor |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+
+|`* *` |tutor |measure the retention rate of my tutoring service |improve on myself and use it as my tuition credential
+
+|`* *` |user |track my work hours over a specific interval of time |maintain a good work life balance
+
+|`* *` |user |receive payments for my services electronically |receive payment more conveniently
+
+|`* *` |tutor |track the tuition payment |know who has not paid
+
+|`* *` |tutor |integrate my calendar with the school's calendar |prevent clashes between tuition schedule and students' school commitment
+
+|`* *` |tutor with many students |send messages to all students that meet a certain description |inform students together and not one by one
+
+|`* *` |tutor |add subject grade of the students |keep track of his progress
+
+|`* *` |tutor |list students coming from the same school |so that I can gather and spread useful information among these students.
+
+|`* *` |tutor |list students based on their grades |give more tuition to students who are doing badly
+
+|`* *` |tutor |give assignments(done and to-be-done) to a student |track his work and progress easily
 
 |`*` |user with many persons in the address book |sort persons by name |locate a person easily
+
+|`*` |tutor |take note of the name of my student's school teachers |adjust my teaching according to the school teacher.
+
+|`*` |tutor |insert my student's photo |
 |=======================================================================
 
 _{More to be added}_
 
 [appendix]
 == Use Cases
 
-(For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
+(For all use cases below, the *System* is the `Tuition Connect AddressBook (TC)` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: UC01-Delete person
```

Comment:
Upon rendering, this is not having D.x tag, while the rest use-cases do. Please fix this inconsistency.